README.md

```diff
@@ -202,44 +234,96 @@ the `lookup` result will sometimes provide a new array, and sometimes not.
   supported.
 * **label** [String][]: Optional label in which to store the result.
 
-## `execute()`
+**Returns:** `this<SetupChain>`
 
-Executes each task added sequentially and collects the results into a single object.
-The results for `execute` are returned. The internal state will be Persisted.
+```javascript
+const SetupChain = require('@logdna/setup-chain')
 
-**returns**: [Object][]
+class MyChain extends SetupChain {
+  constructor(state) {
+    super(state, {
+      hello: async () => {
+        return 'hi'
+      }
+    })
+  }
+}
 
-```javascript
-await new SetupChain()
-  .foo()
-  .bar()
-  .insertDBRecord({}, 'record')
+const chain = new MyChain()
+chain
+  .repeat(5, 'hello', {}, 'result')
   .execute()
+  .then((result) => {
+    console.log(result)
+  })
 
-{
-  foo: 'bar'
-, bar: 'foo'
-, record: { id: 1 }
-}
+> {result: ['hi', 'hi', 'hi', 'hi', 'hi']}
 ```
 
-## `sleep(<Object>)`
+## `sleep(opts)`
```

Comment:
I didn't add an example yet here because it seemed like a lot to wire one up for such a simple function.  If we had a `$now` helper, we could easily illustrate `sleep` by calling `now` while sleeping in between.

I think I will add this as a new commit, and this will be a `minor` release instead of just boring doc fixes. Thoughts.

Reply:
Added using `!date`